scripts/lib/wic/canned-wks/cfg

```diff
@@ -1,6 +1,6 @@
 serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
 
-timeout=5
+timeout=1
```

Comment:
I actually do not think this is that interesting. Not something to take into account when measuring boot time.

If we want to reduce it for demonstration purposes we should probably just remove the menu altogether. If I remember correctly one can get the menu to show during boot by pressing ESC.

Reply:
Yes, I completely agree. This is not a practical optimization of any kind. It is theoretically possible to gain access to the boot menu even if `timeout=0`, but I did not manage to do so :( I can drop this change if someone insists.

Reply:
I am not against it per se but think we should remove it completely if we want a "smooth experience (tm)". Pressing ESC to get it to show does not work? Maybe do this change now and add a sub-task to the boot time task for investigating later if you can not get it to show?